modules/packages/Sort.chpl

```diff
@@ -1617,6 +1690,44 @@ module ShallowCopy {
 
   // TODO: move these out of the Sort module and/or consider
   // language support for it.
+  inline proc shallowCopy(ref dst:?t, ref src:t) {
```

Comment:
I don't see an issue on this TODO - we should open an issue on this sometime soon so it doesn't go forgotten.

Reply:
I created issue #14576.